crates/ty_python_semantic/resources/mdtest/generics/legacy/classes.md

```diff
@@ -319,7 +319,7 @@ class C(Generic[T, U]):
 class D(C[V, int]):
     def __init__(self, x: V) -> None: ...
 
-reveal_type(D(1))  # revealed: D[Literal[1]]
+reveal_type(D(1))  # revealed: C[V, int]
```

Comment:
We probably shouldn't let `V` leak here, it should probably specialize to `Unknown`? That might be out of scope for this PR, but let's at least add a TODO comment